packages/design-system/src/types/FormLabel/FormLabel.d.ts

```diff
@@ -51,6 +51,9 @@ export interface FormLabelProps {
   textClassName?: string;
 }
 
-export default class FormLabel extends React.Component<FormLabelProps, any> {
+export default class FormLabel extends React.Component<
+  FormLabelProps, 
```

Comment:
We need to extend the React.HTMLProps here, other wise passing in an attribute not defined in `FormLabelProps` will cause an error, i.e. `aria-label`